components/Feature/TopicExplorer/index.js

```diff
@@ -18,6 +18,19 @@ const TopicExplorer = (props) => {
     setSearchResults(results);
   };
 
+  const populateInput = event => {
+    const term = event.target.attributes.getNamedItem('data-search-term').value;
+    var results = []
+    const newSearchTerm = term.toLowerCase();
+
+    for(const topic of props.topics) {
+      if(topic['tags'].includes(newSearchTerm)) {
+        results.push(topic.prompt);
+      }
+    }
```

Comment:
I know it's only the first repeat, but would it be good to DRY up this filtering code? (it's repeated pretty similarly just above)

Reply:
This has been refactored into its own method